operator/build/build_kvmoperator.sh

```diff
@@ -32,27 +32,27 @@ fi
 # remove old images
 docker images | grep kvmsoperator | awk '{print $3}' | xargs -I {} docker rmi -f {} 2> /dev/null
 
-echo "[INFO] Removed existing kvmsoperator/kvmsoperator images"
+echo "[INFO] Removed existing accuknox/kvmsoperator images"
```

Comment:
```suggestion
echo "[INFO] Removed existing kubearmor/kvmsoperator images"
```

Reply:
But will we be using kubearmor repo or accuknox repo ?